packages/block-library/src/search/style.scss

```diff
@@ -12,6 +12,8 @@ $button-spacing-y: math.div($grid-unit-15, 2); // 6px
 	svg {
 		min-width: $grid-unit-30;
 		min-height: $grid-unit-30;
+		width: 1.25em;
+		height: 1.25em;
```

Comment:
🏆  to @andrewserong for suggesting this.


https://github.com/WordPress/gutenberg/assets/6458278/c4ccf5a8-bb07-46d9-9d6b-34f222f3ea39